### PR TITLE
Update Profile (user) and Account (user) to include additional fields.

### PR DESCRIPTION
**Pull request by toyhammered**
# What
Update the Profile and Account to have additional Fields. For the Profile, add additional relations including followers, followings, library_entries, and pinned post.

# Why
This is basic functionality that needs to exist. Currently have no filtering based on status which we most likely want to add. Also, this is quite slow to load all library_entries for a user based on a media_type....

# Checklist

- [ ] All files pass Rubocop
- [ ] Any complex logic is commented
- [ ] Any new systems have thorough documentation
- [ ] Any user-facing changes are behind a feature flag (or: explain why they can't be)

- [ ] All the tests pass
- [ ] Tests have been added to cover the new code

### CODE REVIEW

File: app/graphql/types/library_entry/status.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Types::LibraryEntry::Status < Types::BaseEnum
-  value 'current', ''
-  value 'planned', ''
-  value 'completed', ''
-  value 'on_hold', ''
-  value 'dropped', ''
+  value 'current', 'Media that you currently read/watch.'
```
Comment (wopian) [inline app/graphql/types/library_entry/status.rb:2]: `that you are currently reading/watching.`

Comment (wopian) [reply app/graphql/types/library_entry/status.rb:2]: Although can probably strip `that` from all of these since it's a bit redundant.

Comment (toyhammered) [reply app/graphql/types/library_entry/status.rb:2]: yeah, you are right, thank you!

**Comment by toyhammered** [comment]
Going to be making a bunch of smaller PRs

**Pull request closed without merging**